### ISSUE TITLE
Add Monoscope Pattern

### ISSUE THREAD

**Issue by blitzcode** [filipalac/240pTestSuite-PS1#14]
Would you please consider adding the Monoscope Pattern to your version of the 240p testsuite?

https://junkerhq.net/xrgb/index.php?title=240p_test_suite#Monoscope

Thanks!

**Comment by ArtemioUrbina** [comment]
As a note the monoscope pattern needs to be designed for each conole Pixel Aspect Ratio, it wouldn't work if just grabbed form other version.

**Comment by blitzcode** [comment]
For the PS1 I'd assume the preferred resolution would be 320x240 and the existing implementations are probably all either 256 wide or 224 high. That would indeed make it harder than just adding another menu entry and displaying the image :/

**Comment by ArtemioUrbina** [comment]
It is not just the resolution, you need to calculate the Picture Aspect Ratio from the video clocks. We have 320x240 versions on Wii and Dreamcast, but they won't work directly du eto teh above factors. That's why I haven't released all versions yet, each one takes a lot of time by Keith Raney and myself, since we test them for accuracy to our best

**Comment by blitzcode** [comment]
Ah, so because the front/back porch, sync intervals etc. have an effect on the size of the active picture area you couldn't correctly adjust hsize/vsize/aspect if you didn't tweak the pattern for each console? I'm surprised you even have enough precision in a 320x240 image to tweak the squares for such minute differences!

I was mostly interested in the pattern because it seems ideal for tweaking static & dynamic convergence and I prefer the PS1 version cause a PSone console is so small and portable.

**Comment by ArtemioUrbina** [comment]
Yes, that's the case. There's always a small error, but Keith has become an expert in designing them and they are tested against his Kenwood CG-961 NTSC Monoscope Pattern Generator calibrated monitors

**Comment by blitzcode** [comment]
Sounds pretty hardcore! Well, I'll be patient and wait for an update.

**Comment by ArtemioUrbina** [comment]
Just as a note, I am not the developer of this version. Was just adding the info on what is needed.

**Comment by blitzcode** [comment]
Sure. Seems like the bulk of the work would be making the PS1 timings adjusted version of the pattern, though.

**Comment by Takehaniyasubiko** [comment]
Any update on this?

The PS1 version is really handicapped without the monoscope pattern.
